transitions/index.js

```diff
@@ -214,40 +220,33 @@ var applyTransition = function(options, callback) {
 
     /*
      * Transitions are async and should return true if the document has
-     * changed.  If a transition errors then log the error, but don't
-     * return it because that will stop the series and the other
-     * transitions won't run.
+     * changed.  If a transition errors then log the error, but don't return it
+     * because that will stop the series and the other transitions won't run.
      */
     transition.onMatch(change, db, audit, function(err, changed) {
         logger.debug(
-            'finished transition for doc %s seq %s transition %s',
-            change.id, change.seq, key
+            'finished transition %s for seq %s doc %s is %s',
+            key, change.seq, change.id, changed ? 'changed' : 'unchanged'
         );
-        // todo?
-        // prop = doc.transitions && doc.transitions[key] ? doc.transitions[key] : {};
-        //_setProperty('tries', prop.tries ? prop.tries++ : 1)
-        //_setProperty('couch', 'uuid');
-        //
         if (err || changed) {
```

Comment:
We shouldn't even store `err` - the `hasRun` check isn't sophisticated enough to distinguish between successes and failures. Change this to just `if (changed) {`